I'm getting a type error when building:
```
src/main.tsx:6:21 - error TS2345: Argument of type 'HTMLElement | null' is not assignable to parameter of type 'Element | DocumentFragment'.
  Type 'null' is not assignable to type 'Element | DocumentFragment'.

6 ReactDOM.createRoot(document.getElementById('root')).render(
```

--- a/packages/create-vite/template-react-ts/package.json
+++ b/packages/create-vite/template-react-ts/package.json
@@ -12,7 +12,7 @@
     "react-dom": "^18.0.0"
   },
   "devDependencies": {
-    "@types/react": "17.0.43",
+    "@types/react": "^17.0.43",
     "@types/react-dom": "^17.0.14",
     "@vitejs/plugin-react": "^1.3.0",
     "typescript": "^4.6.3",

--- a/packages/create-vite/template-react-ts/src/main.tsx
+++ b/packages/create-vite/template-react-ts/src/main.tsx
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom/client'
 import App from './App'
 import './index.css'
 
-ReactDOM.createRoot(document.getElementById('root')).render(
+ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
     <App />
   </React.StrictMode>

